README.md

```diff
@@ -1,6 +1,10 @@
 # SEM Size Analysis
 
-_A tool for segmenting SEM images for the purposes of analyzing size distributions of nanoparticles
+SAEMI is a tool for obtaining a Size Analysis of particles in Electron Microscopy Images. It uses an image segmentation model 
```

Comment:
Explicitly state what SAEMI stands for, as you would when defining the term in a paper, otherwise people will keep trying to guess, and have a hard time remembering. People are more likely to use packages that are easier to remember the names of.

Reply:
Also, if you're gonna go with this name, why not name your project after it, so you can do something like `pip install saemi`?